.github/workflows/ci.yml

```diff
@@ -158,7 +158,13 @@ jobs:
         run: script/setup/install-dev-tools
         working-directory: src/github.com/containerd/containerd
 
-      - name: Make
+      - name: Check fmt
+        working-directory: src/github.com/containerd/containerd
+        run: |
+          export PATH=$PATH:$(go env GOPATH)/bin
```

Comment:
We update `PATH` in `Set env` step above:
https://github.com/containerd/containerd/pull/4611/files#diff-e9f950f17198d3d5e3122a44230a09b9R137

I think this line should not be needed.

Reply:
Removed, let's see what ci says :)